solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -637,6 +649,23 @@ pub struct SendTransfer<'info> {
     system_program: Program<'info, System>,
 }
 
+#[derive(Accounts)]
+#[instruction(new_length: usize)]
+pub struct ReallocAccounts<'info> {
+    #[account(mut)]
+    sender: Signer<'info>,
+
+    /// The account holding private IBC storage.
+    #[account(mut, realloc = new_length, realloc::payer = sender, realloc::zero = false, seeds = [SOLANA_IBC_STORAGE_SEED], bump)]
+    storage: Account<'info, storage::PrivateStorage>,
```

Comment:
Should this be Option? Same with `chain`? Otherwise we always resize both accounts to the same size?

Reply:
i think it would be better to have those accounts same size

Reply:
Why would we want them the same size? They store completely different data.

Reply:
well thats true, i just thought we could remain consistent but ig there seems like no point.

Reply:
i added an account instead like how u mentioned.